cuda/gen_babeltrace_cuda_model.rb

```diff
@@ -31,9 +32,9 @@
   "int64_t" => 64,
   "uintptr_t" => 64,
   "size_t" => 64,
-  "CUtexObject" => 64,
-  "CUsurfObject" => 64,
-  "CUdeviceptr" => 64,
+  "CUtexObject_v1" => 64,
```

Comment:
Does that mean that old CUDA will not work anymore? I don't care just for my personal education :D

Reply:
No this is just a transitivity issue, these types are now defined with the `_v1` type that didn't existed before. the versions without suffix still exist. This is the reason I had to add all those transitive closures, typdef on typedefs...